core/src/main/scala/org/apache/spark/metrics/ExecutorMetricType.scala

```diff
@@ -137,7 +138,9 @@ case object GarbageCollectionMetrics extends ExecutorMetricType with Logging {
 
   override private[spark] def getMetricValues(memoryManager: MemoryManager): Array[Long] = {
     val gcMetrics = new Array[Long](names.length) // minorCount, minorTime, majorCount, majorTime
```

Comment:
One last thing - should we update or remove this comment? it's missing the 5th name.
No big deal

Reply:
remove the comment, since it's easy to get the meaning of `names`, kind of trouble to update, how about current